db/migrations/20200329134301_create_median_diss_batch.sql

```diff
@@ -0,0 +1,25 @@
+-- +goose Up
+CREATE TABLE maker.median_diss_batch
```

Comment:
Looks like this needs an updated timestamp

Reply:
Yup! I was just about to do that :) thanks for looking out though